src/main/java/oc/wh40k/units/im/IMSpaceWolvesRuestkammer.java

```diff
@@ -13,6 +16,8 @@
     String default2 = "";
     boolean character = false;
     
+    List<String> characters = (List<String>) Arrays.asList("Rune Priest", "Wolf Priest", "Wolf Lord", "Wolf Guard Battle Leader", "Primaris Battle Leader", "Iron Priest", "Primaris Ancient", "Great Company Ancient", "Great Company Champion");
```

Comment:
Declare as constant, use an immutable Set:

```java
Set<String> CHARACTERS = ImmutableSet.of("Rune Priest", "Wolf Priest", "Wolf Lord", "Wolf Guard Battle Leader", "Primaris Battle Leader", "Iron Priest", "Primaris Ancient", "Great Company Ancient", "Great Company Champion");
```

Sets have a faster lookup for `contains` and better represent wha you are trying to express. The order does not matter here.

Reply:
Done. Didn't exactly know how to use the Set interface by itself nor the fastest lookup type.